config/initializers/apipie.rb

```diff
@@ -19,13 +19,7 @@
   config.default_locale = FastGettext.default_locale
   config.locale = ->(loc) { loc ? FastGettext.set_locale(loc) : FastGettext.locale }
 
-  config.translate = lambda do |str, loc|
-    old_loc = FastGettext.locale
-    FastGettext.set_locale(loc)
-    trans = _(str) if str
-    FastGettext.set_locale(old_loc)
-    trans
-  end
+  config.translate = ->(stc, loc) { str ? FastGettext.with_locale(loc) { _(str) } : nil }
```

Comment:
```suggestion
  config.translate = ->(str, loc) { str ? FastGettext.with_locale(loc) { _(str) } : nil }
```

Reply:
Argh, I had that fixed on my laptop, then rebased on my desktop where I made the mistake in the first place ...